conda/models/version.py

```diff
@@ -243,25 +243,29 @@ def __init__(self, vstr):
                     # strings in phase => prepend fillvalue
                     v[k] = [self.fillvalue] + c
 
-    def __str__(self):
+    def __str__(self) -> str:
         return self.norm_version
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return f'{self.__class__.__name__}("{self}")'
 
-    def _eq(self, t1, t2):
+    def _eq(self, t1: list[str], t2: list[str]) -> bool:
         for v1, v2 in zip_longest(t1, t2, fillvalue=[]):
             for c1, c2 in zip_longest(v1, v2, fillvalue=self.fillvalue):
                 if c1 != c2:
                     return False
         return True
 
-    def __eq__(self, other):
+    def __eq__(self, other: object) -> bool:
```

Comment:
Wouldn't `Any` be better choice here? I'm wondering why you went with object.

Reply:
LGTM as the input and not the return type: https://stackoverflow.com/questions/39817081/typing-any-vs-object

Reply:
hm interesting, well I'm all for replacing `Any` with `object` throughout conda, one less import